### PR TITLE
Added a server config option to use a different HTTPS port

### PR DESCRIPTION
**Pull request by blujai831**
This pull request would allow users to use a different HTTPS port by setting `sslPort` in `serverConfig.toml`.

### CODE REVIEW

File: HOW-TO-RUN-A-SERVER.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# How to run a Server
+[# How to run a Server
```
Comment (Wavesonics) [inline HOW-TO-RUN-A-SERVER.md]: Looks like the `[` may have been added by mistake, could you remove it?

**Review by Wavesonics — CHANGES_REQUESTED** [review]
Looks good, and thanks for the contribution!

Just one small change and it'll be good to merge.

File: HOW-TO-RUN-A-SERVER.md (hunk 1 above)
Comment (blujai831) [reply HOW-TO-RUN-A-SERVER.md]: Whoops. Now how did that get in there!

**Comment by Wavesonics** [comment]
Looks good, thanks for the contribution!

**Pull request merged**